msal/managed_identity.py

```diff
@@ -223,7 +223,7 @@ def __init__(
         you may use an environment variable (such as MY_MANAGED_IDENTITY_CONFIG)
         to store a json blob like
         ``{"ManagedIdentityIdType": "ClientId", "Id": "foo"}`` or
-        ``{"ManagedIdentityIdType": "SystemAssignedManagedIdentity", "Id": null})``.
+        ``{"ManagedIdentityIdType": "SystemAssigned", "Id": null}``.
```

Comment:
The detection logic is to compare `ManagedIdentityIdType`'s value with `"SystemAssigned"`:

https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/6e58a3f168165014c3f303c491bb89ada1b845a0/msal/managed_identity.py#L57

https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/6e58a3f168165014c3f303c491bb89ada1b845a0/msal/managed_identity.py#L39

Using `"SystemAssignedManagedIdentity"` will trigger error:

https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/6e58a3f168165014c3f303c491bb89ada1b845a0/msal/managed_identity.py#L239-L240